specification/authorization/resource-manager/Microsoft.Authorization/preview/2017-10-01-preview/authorization.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "AuthorizationManagementClient",
-    "version": "2015-07-01",
```

Comment:
>"2015-07-01", [](start = 15, length = 13)

the earlier api version is correct,since we added new api-version support for roleassignment calls only

Reply:
@mcardosos what about the above comment?